### PR TITLE
Fix typo and grammatical errors in community docs and dynamo docs

### PR DESCRIPTION
**Pull request by ajithvcoder**
Fixes typo and grammatical errors in community docs and dynamo docs

### CODE REVIEW

**Review by H-Huang — CHANGES_REQUESTED** [review]
Thanks! The grammar changes don't need to be added, will accept and merge the typo changes if those are removed

File: docs/source/community/design.rst [hunk 1]
```diff
@@ -92,7 +92,7 @@ model resources with very different performance characteristics
 uniformly, the details will leak) and the `End-to-End
 Principle <http://web.mit.edu/Saltzer/www/publications/endtoend/endtoend.pdf>`__
 (TLDR: building smarts into the lower-layers of the stack can prevent
-building performant features at higher layers in the stack, and often
```
[the remark this answers is not in the document]
Comment (ajithvallabai) [reply docs/source/community/design.rst]: done

File: docs/source/dynamo/faq.rst [hunk 2]
```diff
@@ -19,7 +19,7 @@ Do you support Distributed code?
 --------------------------------
 
 DDP has been tested and works, support for other distributed training
-libraries is under discussion.
```
[the remark this answers is not in the document]
Comment (ajithvallabai) [reply docs/source/dynamo/faq.rst]: done

File: docs/source/dynamo/get-started.rst [hunk 3]
```diff
@@ -23,7 +23,7 @@ vector. A more famous pointwise op you might want to use would
 be something like ``torch.relu()``. Pointwise ops in eager mode are
 suboptimal because each one would need to read a tensor from
 memory, make some changes, and then write back those changes. The single
-most important optimization that inductor does is fusion. So back to our
```
[the remark this answers is not in the document]
Comment (ajithvallabai) [reply docs/source/dynamo/get-started.rst]: done

**Review by H-Huang — APPROVED** [review]
LGTM

**Pull request closed without merging**